### ISSUE TITLE
Generate grain IDs automatically

### ISSUE THREAD

**Issue by 0x70b1a5** [uqbar-dao/ziggurat-ui#42]
When building a grain, we can actually generate the ID automatically based on the input values (lord/holder/town/salt). I will make a PR on the backend that adjusts this behavior in the %add-to-state and %add-test-expectation pokes, such that you don't need to provide an ID field. Will tag someone once that change is merged into master, such that we can prepare a matching frontend release.
